### PR TITLE
PXD-1310 Fix/css refactor 3

### PR DESCRIPTION
**Pull request by qingyashu**

### CODE REVIEW

File: src/components/tables/AccessTable.less [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.access-table .base-table__cell:nth-child(1),
```
Comment (abgeorge7) [inline src/components/tables/AccessTable.less]: can we just do a generic `text-align: left` ? I think there's only two columns

File: src/components/tables/KeyTable.less [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.key-table .base-table__cell:nth-child(1),
```
Comment (abgeorge7) [inline src/components/tables/KeyTable.less]: same here about `text-align: left` for all

File: src/components/tables/base/Table.less [hunk 3]
```diff
@@ -0,0 +1,46 @@
+.base-table__body {
+  border-collapse: collapse;
+  overflow: auto;
+  margin: 1em 0em;
+  text-align:center;
+  width:100%;
```
Comment (abgeorge7) [inline src/components/tables/base/Table.less]: this line and line above - spacing

File: src/components/tables/base/Table.less [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.base-table__head {
```
Comment (abgeorge7) [inline src/components/tables/base/Table.less]: I think this is the same as `.base-table__foot`

File: src/components/tables/base/TableRow.jsx [hunk 5]
```diff
@@ -1,60 +1,23 @@
...
-      );
-    }
     return (
-      <TRow>
-        {
-          this.renderCols(this.props.cols)
-        }
-      </TRow>
-    );
+      <tr className="base-table__row base-table__row--strip-color">
```
Comment (abgeorge7) [inline src/components/tables/base/TableRow.jsx]: think this should be `.base-table__row--stripe-color`

File: src/Submission/SubmitForm.less [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+.submit-form__input {
+  width: 400px;
+  height:40px;
+  padding: 0px 10px;
+}
+
+.submit-form__label {
+  margin: 3px;
+  display:inline-block;
```
[the remark this answers is not in the document]
Comment (qingyashu) [reply src/Submission/SubmitForm.less]: Thanks!

File: src/components/tables/AccessTable.less (hunk 1 above)
Comment (qingyashu) [reply src/components/tables/AccessTable.less]: I agree!!! 😂

File: src/components/tables/TransactionLogTable.less [hunk 7]
```diff
@@ -0,0 +1,12 @@
+.transaction-log-table__status-bar {
+  font-size: 16px;
+  text-align: left;
```
Comment (abgeorge7) [inline src/components/tables/TransactionLogTable.less]: This is only applying to `Succeeded` states:
screen shot 2018-08-06 at 10 27 21 am

File: src/components/tables/TransactionLogTable.less [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+.transaction-log-table__status-bar--fail {
+  color: #ff2200;
```
Comment (abgeorge7) [inline src/components/tables/TransactionLogTable.less]: Also it doesn't look like the color is being applied to the text

File: src/components/tables/TransactionLogTable.jsx [hunk 9]
```diff
@@ -28,11 +15,11 @@ class TransactionLogTable extends Component {
   };
 
   stateToColor = state => (state === 'SUCCEEDED' &&
-      <StatusBar className='special-number'>{formatText(state)}</StatusBar>)
+      <div className='form-special-number transaction-log-table__status-bar'>{formatText(state)}</div>)
```
[the remark this answers is not in the document]
Comment (qingyashu) [reply src/components/tables/TransactionLogTable.jsx]: This should be 'form-special-number', spec confirmed, previous classname must be typo

[the remark this answers is not in the document]
Comment (abgeorge7) [reply src/components/tables/TransactionLogTable.jsx]: oh okay I didn't see `form-special-number` in our base.less 👍

**Pull request merged**